uknetdev.c

```diff
@@ -699,8 +700,10 @@ struct netif *uknetdev_addif(struct uk_netdev *n
 		return NULL;
 	}
 
-	if (hostname)
+	if (hostname) {
 		netif_set_hostname(nf, hostname);
+		sethostname(hostname, strlen(hostname));
```

Comment:
In `nolibc`'s `<unistd.h>`, the prototype declaration is guarded with `CONFIG_LIBPOSIX_SYSINFO`. The idea behind this that the call is only available if `posix-sysinfo` is part of a configuration. I think we should do the same here.

Reply:
Good call, will update.